mappings/net/minecraft/entity/ai/brain/sensor/Sensor.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS aon net/minecraft/entity/ai/brain/sensor/Sensor
+	FIELD b SENSE_COOLDOWN I
+	METHOD a getRequiredMemoryModules ()Ljava/util/Set;
```

Comment:
Required Modules is correct, but I think it's more accurately described as Output Modules